src/Shared/Recruit.Shared.Web/ViewModels/Skills/ISkillsEditModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Esfa.Recruit.Shared.Web.ViewModels.Skills
+{
+    public interface ISkillsEditModel
```

Comment:
We have a couple of interfaces like this that we would like to remove. They are not contracts for behaviour but state and are best avoided around dto's. Can we not use an abstract class instead.

Reply:
The differing VacancyRouteModels prevent this.  I'll have another rethink though and see if I can get rid of these interfaces.

Reply:
I've removed the inheritance from VRM